modules/eks/variables.tf

```diff
@@ -42,6 +42,7 @@ variable "node_pools" {
     type              = optional(string, "self-managed") # "eks-managed" or "self-managed"
     name              = string
     ami_id            = optional(string)
+    default_ami_type  = optional(string, null)
```

Comment:
```suggestion
    default_ami_type  = optional(string, null) # null to use the value of var.global_eks_nodepool_default_ami_type
```

Reply:
I'd call this variable `ami_type`, `default_ami_type` suggests that you could change it individually and is redundant.

https://www.youtube.com/watch?v=PEgk2v6KntY

Reply:
Ok, it make sense. However should be a good idea to clarify that `ami_id` and `ami_type` are mutually exclusive

Reply:
ami_type implemented